RankUpper-56/src/main/java/br/net/fabiozumbi12/RankUpper/config/RUConfig.java

```diff
@@ -188,44 +189,44 @@ public boolean checkRankup(User p){
             Optional<PlaceholderService> phapiOpt = Sponge.getServiceManager().provide(PlaceholderService.class);
             if (phapiOpt.isPresent()) {
                 PlaceholderService phapi = phapiOpt.get();
-                for (Entry<String, Long> key:root.ranked_groups.get(pgroup).placeholder_api_requirements.entrySet()){
-                    Optional<Long> optVal = phapi.parse(key.getKey(), p.getPlayer().isPresent() ? p.getPlayer().get(): p, null, Long.class);
-                    if (key.getValue() > 0 && optVal.isPresent()){
-                        if (optVal.get() < key.getValue()){
+                for (Entry<String, Long> key : root.ranked_groups.get(pgroup).placeholder_api_requirements.entrySet()) {
+                    double optVal = RUUtil.getPlaceholderValue(phapi, key.getKey(), p.getPlayer().isPresent() ? p.getPlayer().get() : p);
```

Comment:
These were the only changes in this file, the rest if intellij auto formatting the class